SwissTransferCore/UserDefaults+Keys.swift

```diff
@@ -129,4 +131,22 @@ public extension UserDefaults {
             set(newValue, forKey: key(.sentryAuthorized))
         }
     }
+
+    var transferCount: Int {
+        get {
+            integer(forKey: key(.transferCountKey))
+        }
+        set {
+            set(newValue, forKey: key(.transferCountKey))
+        }
+    }
+
+    var hasReviewedApp: Bool {
+        get {
+            bool(forKey: key(.hasReviewedApp))
```

Comment:
You should use the same method as `isSentryAuthorized` for example. With `object`, `set` and then `bool`